javascript/code examples/browser.md

```diff
@@ -0,0 +1,37 @@
+## Browser
+Pushing file to lighthouse node from browser.
+
+```javascript
+import React from "react";
+import axios from "axios";
+import { ethers } from 'ethers';
+import lighthouse from 'lighthouse-web3';
+
+function App() {
+
+  const sign_message = async () => {
+    const provider = new ethers.providers.Web3Provider(window.ethereum);
+    const signer = provider.getSigner();
+    const address = await signer.getAddress();
+    const message = (await axios.get(`https://api.lighthouse.storage/api/auth/get_message?publicKey=${address}`)).data;
+    const signed_message = await signer.signMessage(message);
+    return({
+      signed_message: signed_message,
+      address: address
+    });
+  }
+
+  const deploy = async(e) =>{
```

Comment:
what is **e** here. Use self-explanatory words as names. Also comment before each function describing input and return parameters.